alarm/utils.py

```diff
@@ -0,0 +1,62 @@
+from alarm.binance_utils import extract_binance_data_from_socket, parse_binance_data
+from alarm.models import Candle, Coin
+
+
+def check_if_call_needed(prices):
+    threshold = prices['threshold']
+    last_candle_high_price = prices['last_candle_high_price']
+    current_candle_high_price = prices['current_candle_high_price']
+    last_candle_low_price = prices['last_candle_low_price']
+    current_candle_low_price = prices['current_candle_low_price']
+
+    if min(last_candle_low_price, current_candle_low_price) <= threshold <= max(last_candle_high_price,
+                                                                                current_candle_high_price):
+        result = True
+    else:
+
+        result = False
+
+    return result
+
+
+def get_binance_data_and_update_coin_candle(data):
```

Comment:
I'd call it " update_coin_candle_from_binance_data", or something like this

Reply:
++